drivers/hmis_csv_importer/app/models/hmis_csv_importer/temp_table.rb

```diff
@@ -0,0 +1,44 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HmisCsvImporter
+  class TempTable
+    # temporary table functionality from https://github.com/zdennis/ar-extensions/blob/master/ar-extensions/lib/ar-extensions/temporary_table.rb
```

Comment:
This seems to work but the referenced code is from 15 years ago, I think in modern rails this can be much simpler. I'm not sure that we even need a class for this. And I don't think there's a need to wrap it in an AR model

```ruby
GrdaWarehouseBase.connection.create_table table_name, temporary: true do |t|
  t.references : source, null: false
  t.references : source, null: false
end
```